.circleci/config.yml

```diff
@@ -1,47 +1,44 @@
-version: 2
-jobs:
-  test:
-    docker:
-      - image: artsy/hokusai:0.5.2
-    working_directory: ~/volley
-    steps:
-      - add_ssh_keys
-      - checkout
-      - setup_remote_docker
-      - run:
-          name: Test
-          command: hokusai test
-  deploy:
-    docker:
-      - image: artsy/hokusai:0.5.2
-    environment:
-      - AWS_DEFAULT_REGION: us-east-1
-    steps:
-      - add_ssh_keys
-      - checkout
-      - setup_remote_docker
-      - run:
-          name: Push
-          command: hokusai registry push --tag $CIRCLE_SHA1 --force --overwrite
-      - run:
-          name: Configure
-          command: hokusai configure --kubectl-version 1.10.7 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
-      - run:
-          name: Validate Kubernetes Yaml
-          command: |
-            kubectl --context staging apply -f ./hokusai/staging.yml --dry-run
-      - run:
-          name: Deploy
-          command: hokusai staging deploy $CIRCLE_SHA1
+version: 2.1
+
+orbs:
+  hokusai: artsy/hokusai@0.4.0
```

Comment:
The current version is 0.7.0, is it intentional to use a lower one?

Reply:
Ach good catch!  No I just copied from setup templates - should update those as well!